views/includes/layout.jade

```diff
@@ -2,59 +2,48 @@ doctype html
-    crossorigin="anonymous" )
 
+    //-     Styles      -//
+    block styles
+        link(rel="stylesheet", href="/node_modules/bootstrap/dist/css/bootstrap.min.css")
+        link(rel="stylesheet", href="/node_modules/bootstrap/dist/css/bootstrap-theme.min.css")
+        if (language === 'he')
+            link(rel="stylesheet", href="/node_modules/bootstrap-rtl/dist/css/bootstrap-rtl.css")
+        link(rel='stylesheet', href='/node_modules/font-awesome/css/font-awesome.min.css')
+        link(rel='stylesheet', href='/node_modules/ui-select/dist/select.min.css')
+        link(rel='stylesheet', href='/node_modules/selectize.js/dist/css/selectize.default.css')
+        link(rel="stylesheet", href="/stylesheets/main.scss" )
+
+    //-     Scripts     -//
+    script(src='/node_modules/jquery/dist/jquery.min.js')
+    script(src="/node_modules/bootstrap/dist/js/bootstrap.min.js")
+    script(src='/node_modules/bootstrap-validator/dist/validator.min.js')
+    script(src='/node_modules/bootstrap-table/dist/bootstrap-table.min.js')
     if (language === 'he')
-        link( rel="stylesheet" href="https://cdn.rawgit.com/morteza/bootstrap-rtl/v3.3.4/dist/css/bootstrap-rtl.min.css")
-
-    link( rel="stylesheet", href="/stylesheets/main.scss" )
-    // Font-awesome
-    link(href='/bower_components/gentelella/vendors/font-awesome/css/font-awesome.min.css', rel='stylesheet')
-    // SweetAlert
-    link(rel='stylesheet', type='text/css', href='/bower_components/sweetalert/dist/sweetalert.css')
-    // ng-ui-select
-    link(rel='stylesheet', type='text/css', href='/bower_components/angular-ui-select/dist/select.min.css')
-    link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/selectize.js/0.8.5/css/selectize.default.css')
-
-body( style='direction:#{languageDir}', lang=language, dir='#{languageDir}' )
-    block wrapper
-    //- Scripts
-    script(src='/bower_components/gentelella/vendors/jquery/dist/jquery.min.js' )
-    script(src="/bower_components/gentelella/vendors/bootstrap/dist/js/bootstrap.min.js",
-    integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS",
-    crossorigin="anonymous" )
-    script(async src='https://cdnjs.cloudflare.com/ajax/libs/1000hz-bootstrap-validator/0.11.5/validator.min.js' )
-    script(async src='/bower_components/bootstrap-table/dist/bootstrap-table.min.js')
-    if (language === 'he')
-        script(async src='/bower_components/bootstrap-table/dist/locale/bootstrap-table-he-IL.min.js')
+        script(src='/node_modules/bootstrap-table/dist/locale/bootstrap-table-he-IL.min.js')
 
     // External packages
-    script(async src="https://www.google.com/recaptcha/api.js?hl=#{language}")
-    script(src='/bower_components/sweetalert/dist/sweetalert.min.js')
-    script(src='/bower_components/js-cookie/src/js.cookie.js')
-
+    script(src="https://www.google.com/recaptcha/api.js?hl=#{language}")
```

Comment:
not sure how to change the module with language parameter
node_module/recaptcha/lib/recaptcha.js doesn't have a constructor with only one param language.

In addition, I think we should move this script to the signup.jade page, since we only need it there...
doesn't make sense to import it in every page.